raiden/smart_contracts/NettingChannelLibrary.sol

```diff
@@ -112,36 +119,44 @@ library NettingChannelLibrary {
 
     /// @notice partner() to get the partner or other participant of the channel
     /// @dev Get the other participating party of the channel
-    /// @return partnerAddress (address) the partner of the calling party
+    /// @return (address) the partner of the calling party
     function partner(Data storage self, address one_address) constant returns (address) {
         Participant[2] storage participants = self.participants;
         Participant storage node1 = participants[0];
         Participant storage node2 = participants[1];
 
-        if (one_address == node1.nodeAddress) {
-            return node2.nodeAddress;
+        if (one_address == node1.node_address) {
+            return node2.node_address;
         }
 
-        if (one_address == node2.nodeAddress) {
-            return node1.nodeAddress;
+        if (one_address == node2.node_address) {
+            return node1.node_address;
         }
 
         return 0x0;
     }
 
-    function addressAndBalance(Data storage self) constant returns (address participant1, uint balance1, address participant2, uint balance2) {
+    function addressAndBalance(Data storage self)
```

Comment:
I can live with that, I just followed the suggested guidelines from the solidity documentation like this:
```
function thisFunctionNameIsReallyLong(address x, address y, address z)
    public
    onlyowner
    priced
    returns (address)
{
    doSomething();
}

function thisFunctionNameIsReallyLong(
    address x,
    address y,
    address z,
)
    public
    onlyowner
    priced
    returns (address)
{
    doSomething();
}
```
But I am open to making some changes that makes it a bit more compact like you suggest.